src/client/hooks/useCountryIso.ts

```diff
@@ -2,16 +2,16 @@ import { useMatch, useParams } from 'react-router-dom'
 
 import { CountryIso } from '@meta/area'
 
-import { useIsHome, useIsLogin } from '@client/hooks/useIsPath'
+import { useIsCycleLanding, useIsLogin } from '@client/hooks/useIsPath'
 
 export const useCountryIso = (): CountryIso => {
   const { countryIso } = useParams<{ countryIso: CountryIso }>()
-  const match = useMatch(':countryIso/*')
+  const match = useMatch('assessments/:assessmentName/:cycleName/:countryIso/*')
 
   const isLogin = useIsLogin()
-  const isHome = useIsHome()
+  const isHome = useIsCycleLanding()
 
   if (isHome || isLogin) return null
 
-  return countryIso ?? (match.params.countryIso as CountryIso)
+  return countryIso ?? (match?.params.countryIso as CountryIso) ?? null
```

Comment:
This might cause crashes, I would not return null here?
Is it required?

Reply:
there is a return null on line 14, useCountryIso can return null